apps/datahub/src/styles.css

```diff
@@ -35,3 +35,10 @@ html.record-page-active {
     @apply items-center justify-center px-2 py-1 bg-primary hover:text-primary text-white rounded transition-all;
   }
 }
+.tippy-box a {
+  @apply text-primary-lighter;
+  text-decoration: underline;
+}
+.tippy-box a:hover {
+  @apply text-primary-lightest;
+}
```

Comment:
Once we create a separate directive/component we should remember to put this into its stylesheet! (to be done later)